### PR TITLE
Proofread docs

### PR DESCRIPTION
**Pull request by DennyHoang**
Which other branches should this be merged with (if any)?

### CODE REVIEW

File: scst-policy/configuring.md [hunk 1]
```diff
@@ -51,25 +51,30 @@ Glob matches against images using semantics similar to golang filepaths. A `**`
...
 ```
 
 ### <a id="cip-authorities"></a> `authorities`
 
-Authorities listed in the `authorities` block of the ClusterImagePolicy can be `key` specifications. Each `key` authority can contain a raw public key, a
-`secretRef` or a `kms` path.
+Authorities listed in the `authorities` block of the ClusterImagePolicy can be
+`key` or `keyless` specifications.
+
+Each `key` authority can contain a raw ECDSA public key, a `secretRef`, or a
```
Comment (elfotografo007) [inline scst-policy/configuring.md]: I am not sure if "raw" is the right term. For me raw is more like the bytes of the key. Maybe "PEM" or "PEM-encoded" is more appropriate.

File: scst-policy/configuring.md [hunk 2]
```diff
@@ -190,110 +166,99 @@ spec:
...
+        -----BEGIN PUBLIC KEY-----
+        MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEhyQCx0E9wQWSFI9ULGwy3BuRklnt
+        IqozONbbdbqz11hlRJy9c7SG+hdcFl9jE9uE/dwtuwU2MqU9T/cN0YkWww==
+        -----END PUBLIC KEY-----
+```
+
+When using the sample policy, run these commands to verify your configuration:
 
-1. Verify that a signed image, validated with a configured public key, starts.
+1. Verify that a signed image, validated with the configured public key, starts.
```
Comment (elfotografo007) [inline scst-policy/configuring.md]: What does it mean that a signed image starts?

File: scst-policy/configuring.md (hunk 1 above)
Comment (DennyHoang) [reply scst-policy/configuring.md]: updated to PEM-encoded

File: scst-policy/configuring.md (hunk 2 above)
Comment (DennyHoang) [reply scst-policy/configuring.md]: I'll change it to "verify that a signed image is admitted" or similar for the other cases as well. It was the terminology originally there and it just meant that the container starts.

**Review by elfotografo007 — APPROVED** [review]

**Pull request merged**